projects/lib/src/engineconfiguration.cpp

```diff
@@ -29,9 +29,9 @@ EngineConfiguration::EngineConfiguration()
 	  m_validateClaims(true),
 	  m_restartMode(RestartAuto),
 	  m_rating(0),
-	  m_restart_score(0),
 	  m_strikes(0),
-	  m_cuteseal(false)
+      m_restart_score(0),
```

Comment:
Unless there were warnings related to ordering, I wouldn't change anything here. (And if these fix warnings, better to isolate in a separate commit.)